src/main.rs

```diff
@@ -100,6 +100,15 @@ fn main() {
                                .number_of_values(1)
                                .takes_value(true))
 
+                          .arg(Arg::with_name("keep_dir")
+                               .help("Keep only files/directories specified as globs")
+                               .long("keep-only")
+                               .value_name("PATH")
+                               .multiple(true)
+                               .number_of_values(1)
+                               .takes_value(true)
+                               .conflicts_with("ignore_dir"))
```

Comment:
I suppose we could have the both options.
For example: keep=foo/*.rs and ignore=foo/bar_*.rs
So having both options could mean: Keep = Keep - (Keep ∩ Ignore)

Reply:
I chose to make it conflict because that matches the behavior of lcov (`--remove` and `--extract` are conflicting options) and it seemed the least surprising, but I'll take a look at what it would take to implement the set difference as you've described. Thanks!

Reply:
Ah, I see! This behavior already works as implemented. I'll remove the `conflicts_with` and add a test for using both keep and ignore